ts-frontend/src/components/Map.tsx

```diff
@@ -156,6 +182,12 @@ export const Map = () => {
     closeConnection("202A", currentUserUUID);
   };
 
+  const DEFAULT_PROFILE_URL =
+    "https://cdn-icons-png.flaticon.com/128/3177/3177440.png";
+  const DEFAULT_MARKER_PIC =
+    "https://res.cloudinary.com/cantacloud2/image/upload/w_40,h_40,c_scale/v1714413297/hbjh5jqrguqtuvmfr7sl.png";
```

Comment:
export/import these constant URIs to and from the `constants` directory. 
this should make the default URIs consistent across the codebase

Reply:
done